ZenPacks/zenoss/Microsoft/MSMQ/modeler/plugins/zenoss/winrm/MSMQQueueMap.py

```diff
@@ -27,26 +23,30 @@ class MSMQQueueMap(WinRMPlugin):
     )
 
     powershell_commands = {
-        'MSMQQueue': 'Get-MsmqQueue | Select Path',
+        'PersistentPrivateAndPublicQueues': 'Get-MsmqQueue | Select Path',
+        'OtherQueues': 'Get-WmiObject Win32_PerfFormattedData_msmq_MSMQQueue | Select Name',
     }
 
+    def _get_queue_names(self, results):
+        q_names = []
+        for q_group_name, q_output in results.iteritems():
```

Comment:
`q_group_name` is not used, so you may use `results.iterivalues()` instead of `results.iteritems()`.